src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -292,10 +355,26 @@ public void onMouseEnterUsername() {
     if (player == null || player.getChatOnly() || usernameLabel.getTooltip() != null) {
       return;
     }
-
     Tooltip tooltip = new Tooltip();
     usernameLabel.setTooltip(tooltip);
-    clanLabel.setTooltip(tooltip);
+
+    Runnable setClanTooltip = new Runnable() {
+      @Override
+      public void run() {
+        if (clanMenu.getTooltip() == null && clan != null) {
+          Tooltip clanTooltip = new Tooltip();
+          clanMenu.setTooltip(clanTooltip);
+          if (clan.getDescription() == null) {
+            clan.setDescription("-");
+          }
+          clanTooltip.setText(i18n.get("clan.clanName") + "\n" + clan.getClanName() + "\n\r" + i18n.get("clan.describtion") + "\n" + clan.getDescription() + "\n\r" + i18n.get("clan.clanMembers") + "\n" + clan.getClanMembers() + "\n\r" + i18n.get("clan.leader") + "\n" + clan.getLeaderName());
```

Comment:
`clan.description` (**p**).

`\n` is a "linefeed", `\r` a "carriage return". I think you just want to use `\n\n`.
However, I prefer not to build UI elements like this because it's difficult/impossible to style; instead, we should create an FXML file (and therefore a specific controller) for the tooltip contents and use separate labels for each item